eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -40,6 +40,11 @@ public bool SetupEnvironment()
                 Console.WriteLine($"Set DotNetEfFullPath: {dotnetEFFullPath}");
                 EnvironmentVariables.Add("DotNetEfFullPath", dotnetEFFullPath);
 
+                // Playwright will download and look for browsers to this directory
+                var playwrightBrowsers = Path.Combine(helixDir, "ms-playwright");
+                Console.WriteLine($"Setting PLAYWRIGHT_BROWSERS_PATH: {playwrightBrowsers}");
+                EnvironmentVariables.Add("PLAYWRIGHT_BROWSERS_PATH", playwrightBrowsers);
```

Comment:
Rather than doing this through an environment variable, can we do this through the config system in place? That is:
* Generate a `playwright.environment.json` as part of the build (similar to what we do for Mvc.Testing)
* Add the file as a source of config.
* Put the BrowsersPath in the contents and wire that up to be passed down to `Playwright.Create`.

That's much easier for us to debug in the future if something changes and it stops working.

Reply:
I'll file an issue for that to do later, that is not as easy as just using the built in environment variable support